source/extensions/filters/http/ext_authz/config.cc

```diff
@@ -35,7 +35,7 @@ Http::FilterFactoryCb ExtAuthzFilterConfig::createFilterFactoryFromProtoWithServ
     const auto client_config =
         std::make_shared<Extensions::Filters::Common::ExtAuthz::ClientConfig>(
             proto_config, timeout_ms, proto_config.http_service().path_prefix(), server_context);
-    callback = [filter_config, client_config,
+    callback = [filter_config = std::move(filter_config), client_config,
```

Comment:
Yea, the shared_prt copy only needs to be done once, which I thinks is already performed at filter ctor `: config_(config)`

Reply:
Oh, that can also be a std::move.

Reply:
Oh, nevermind... the constructor is taking a reference to a shared pointer.